README.md

```diff
@@ -0,0 +1,110 @@
+
+## Features
+
+- **Launch Chivalry 2**: Easily launch Chivalry 2 with custom configurations.
+- **Mod Management**: Download and manage mods for Chivalry 2.
+- **Customization**: Customize the launcher with various settings and preferences.
+
+## Community
+
+Join the [Chivalry 2 Unchained community on Discord](https://discord.gg/chiv2unchained) to connect with other players, get support, and participate in discussions.
+
+## Installation
+
+1. [Download the latest binary from the releases page](https://github.com/Chiv2-Community/C2GUILauncher/releases).
+2. Open the downloaded file to install the C2UL.
+
+### Installation Instructions
+
+- Rename `Chivalry2Launcher.exe` to `Chivalry2Launcher-ORIGINAL.exe` -- This is important. The name is significant.
+- Put the downloaded `Chivalry2Launcher.exe` in place of the old one.
+- Launch the game normally (either from your steam library or from Epic Games Store).
+
+## General Use Instructions
+
+### How to Use the Tool
+
+1. **Open the Launcher**: Run the C2UL from the installed location.
+2. **Select Installation Type**: If the installation type is not auto-detected, select it manually from the provided options.
+3. **Launch Vanilla Game**:
+   - Click the "Launch Vanilla" button to start the unmodded version of Chivalry 2.
+4. **Launch Modded Game**:
+   - If desired, check the "Enable Debug DLLs" option for debugging purposes.
+   - If you want to skip downloading plugins, check the "Disable Plugin Download" option.
+   - Click the "Launch Modded" button to start the modded version of Chivalry 2. The necessary mod files will be downloaded and applied.
+5. **Monitor Downloads**: If you are launching the modded game, you can monitor the download progress of the mod files in the "Downloads" section.
+
+### Troubleshooting
+
+- If an error occurs during the launch process, an error message will be displayed with details. Check the provided information for troubleshooting.
+
+### Note
+
+- The "Launch Vanilla" and "Launch Modded" buttons will be disabled during the launch process to prevent multiple launches.
+
+---
+
+## Developer-Focused App Structure and Details
+
+### Main Components
+
+- **Chivalry2Launchers**: Contains definitions for Vanilla and Modded launchers.
+  - **VanillaLauncher**: Launches the original, unmodified version of Chivalry 2.
+  - **ModdedLauncher**: Launches the modified version of Chivalry 2 with selected mods.
+- **MainWindow**: The main window of the application, handling user interactions.
+  - **Initialize**: Sets up the main window and loads user preferences.
+  - **HandleLaunch**: Manages the process of launching the game, either in vanilla or modded mode.
```

Comment:
`HandleLaunch`, `UpdateMods, `LaunchGame`, and `MonitorProcess` do not exist. I think it'd be best to only list the classes and what they do, since the functions inside them may be in flux for awhile.  Also eventually we'll hopefully start exporting docs from the code in to a doc site

Also, info that's for devs only should go in to CONTRIBUTING.md IMO